x-pack/plugins/enterprise_search/public/applications/applications/components/search_application/docs_explorer/search_ui_components.tsx

```diff
@@ -163,12 +163,13 @@ export const SearchBar: React.FC<InputProps> = ({ additionalInputProps }) => (
 );
 export const PagingInfoView: React.FC<PagingInfoViewProps> = ({ start, end, totalResults }) => (
   <EuiText size="s">
-    <FormattedHTMLMessage
-      tagName="p"
-      id="xpack.enterpriseSearch.searchApplications.searchApplication.docsExplorer.pagingInfo.text"
-      defaultMessage="Showing <strong>{start}-{end}</strong> of {totalResults}"
-      values={{ end, start, totalResults }}
-    />
+    <p>
+      <FormattedMessage
```

Comment:
This is an invalid fix, the HTML inside (the `strong`) tags should be part of the values as actual HTML elements. Not part of the string.

Reply:
Can you clarify more?  My change is that I removed the `FormattedHTMLMessage` and replaced it with `FormattedMessage`.